src/command_modules/azure-cli-maps/azure/cli/command_modules/maps/_params.py

```diff
@@ -0,0 +1,54 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.arguments import CLIArgumentType
+
+from azure.cli.core.commands.parameters import (
+    get_enum_type,
+    get_resource_name_completion_list,
+    resource_group_name_type,
+    tags_type)
+
+from azure.cli.command_modules.maps.validators import validate_account_name
+from azure.mgmt.maps.models.maps_management_client_enums import KeyType
+
+
+def load_arguments(self, _):
+    # Argument Definition
+    maps_name_type = CLIArgumentType(options_list=['--name', '-n'],
+                                     completer=get_resource_name_completion_list(
+                                         'Microsoft.Maps/accounts'),
+                                     help='The name of the Maps Account',
+                                     validator=validate_account_name)
```

Comment:
As your module grows, this will get very cumbersome.  Most modules simply put one entry on one line and disable pylint's "line-too-long" warning.  Up to you. Opinions vary (mine is pretty clearly in favor of long lines)